app/queries/earliest_patient_data_query.rb

```diff
@@ -23,6 +23,7 @@ def call(region)
         AND (patients.assigned_facility_id in :facility_ids OR patients.registration_facility_id in :facility_ids)
     SQL
     return nil if sql.values.first.blank?
-    Time.zone.parse(sql.values.first)
+    # we store timestamps in UTC in the app tables
+    Time.find_zone("UTC").parse(sql.values.first).in_time_zone(Time.zone)
```

Comment:
I think you can instead use [force_timezone](https://github.com/github/github-ds/blob/c5962e5f59ea97924278a01ea8d85edd29aa2386/lib/github/sql.rb#L143-L144) from GitHub::SQL... I believe it would be `force_timezone: :utc` win the constructor above.

Reply:
That's the elegant solution we were looking for ✨

Reply:
Tried this a bit, but couldn't quite get it to do what we wanted. I'm going to stick to this explicit approach for now.
* Parse the timestamp, interpreting it as a UTC timestamp
* Give me that real time in IST (to maintain the contract of this query)

Reply:
Extracted to https://github.com/simpledotorg/simple-server/pull/2570